app/src/ui/lib/conflicts/unmerged-file.tsx

```diff
@@ -415,31 +418,20 @@ function resolvedFileStatusString(
   if (manualResolution === ManualConflictResolution.theirs) {
     return getUnmergedStatusEntryDescription(status.entry.them, branch)
   }
-  return 'No conflicts remaining'
+  return defaultConflictsResolvedMessage
 }
 
-const renderResolvedFileStatusSummary: React.FunctionComponent<{
-  path: string
-  status: ConflictedFileStatus
-  repository: Repository
-  dispatcher: Dispatcher
-  manualResolution?: ManualConflictResolution
+const getResolvedFileStatusSummary = (
```

Comment:
Since this only really returned a short message, I switched this to only return a message so that I could compare strings in the `renderResolvedFile` to determine when to display the "Undo" button.